packages/default/scss/orgchart/_layout.scss

```diff
@@ -0,0 +1,92 @@
+@include exports("orgchart/layout") {
+
+    // OrgChart
+    .k-orgchart {
+        width: 100%;
+        box-sizing: border-box;
+        font-size: $orgchart-font-size;
+        font-family: $orgchart-font-family;
+        line-height: $orgchart-line-height;
+        display: flex;
+        justify-content: center;
+        overflow: auto;
+    }
+
+    .k-orgchart-container {
+        position: relative;
```

Comment:
if this is the "canvas" of the orgchart, then it should have `width` and `height` set to 100%... Not that the width is not going to be 100%, considering this is a block element.